web/src/components/tasks/cannabis/CannabisPriorityRequirements.tsx

```diff
@@ -202,8 +203,7 @@ export const CannabisPriorityRequirements = (props: Props): ReactElement => {
         {(displayMWPriorityType || displayVeteranPriorityType || displaySocialEquityPriorityType) && (
           <>
             <hr />
-            <div className="green-box text-normal padding-2 margin-top-2 bg-primary-extra-light radius-lg">
-              <div className="text-bold">{Config.cannabisPriorityStatus.greenBoxBoldedText}</div>
```

Comment:
This header ended up being the same as the default header for the `<Callout>` component so it didn't need to exist separately. Instead, that field is now removed from the config .json and from the CMS.